packages/coordinator/src/screens/manageAppointmentsScreen/ManageAppointmentsScreenContainer.tsx

```diff
@@ -20,6 +20,7 @@ export default function ManageAppointmentsScreenContainer() {
     getDefaultState()
   );
   const [isLoading, setIsLoading] = useState(false);
+  const [showOnlyNewChanges, setShowOnlyNewChanges] = useState(false);
```

Comment:
Please go over the PR and be consistent with the naming. 
Now we have `showOnlyNew`, `ChangeType`, `BookingChange` and they all refer to the same thing

Reply:
showOnlyNew converted to showOnlyRecentChanges,
in consistency with RecentChangeType.

BookingChange refers to both recent and non-recent changes